eventify/consumer.py

```diff
@@ -14,7 +14,9 @@ def handler(count):
             print("event received: {0}", count)
 
         try:
-            yield self.subscribe(handler, u'mytopic')
+            topic = self.config.extra['publish_topic']['topic']
```

Comment:
yeah these changes have already been committed - my apologies on not pushing earlier